packages/public/server/src/module/Ui/templates/layout/default/serlo-home.phtml

```diff
@@ -154,36 +156,42 @@ echo $this->doctype();
                             <div class="hidden-md hidden-lg">
                                 <a href="/community" class="home-emph-link">
-                                    <?php echo $this->translate('Visit the landing page for authors'); ?> <i class="fa fa-arrow-circle-right"></i>
+                                    <?php echo $this->translate('Visit the landing page for authors'); ?> <i
+                                        class="fa fa-arrow-circle-right"></i>
                                 </a>
                             </div>
                         </div>
-                        <div class="col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-2 col-lg-offset-1 col-lg-3 home-calls-column">
+                        <div
+                            class="col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-2 col-lg-offset-1 col-lg-3 home-calls-column">
                             <p>
                                 <?php echo $this->translate('We offer a diverse range of jobs and volunteering opportunities in the fields of software development,
                                  design, translation, communications, project management and more.'); ?>
                             </p>
                             <div class="hidden-md hidden-lg">
                                 <a href="/get-involved" class="home-emph-link">
-                                    <?php echo $this->translate('Get involved!'); ?> <i class="fa fa-arrow-circle-right"></i>
+                                    <?php echo $this->translate('Get involved!'); ?> <i
+                                        class="fa fa-arrow-circle-right"></i>
                                 </a>
                             </div>
                         </div>
                     </div>
                     <div class="home-row home-column visible-md visible-lg">
                         <div class="col-md-4 col-md-offset-1 col-lg-3 col-lg-offset-2 home-calls-column">
                             <a href="/community" class="home-emph-link">
-                                <?php echo $this->translate('Visit the landing page for authors'); ?> <i class="fa fa-arrow-circle-right"></i>
+                                <?php echo $this->translate('Visit the landing page for authors'); ?> <i
+                                    class="fa fa-arrow-circle-right"></i>
                             </a>
                         </div>
                         <div class="col-md-4 col-md-offset-2 col-lg-offset-1 col-lg-3 home-calls-column">
                             <a href="/get-involved" class="home-emph-link">
-                                <?php echo $this->translate('Get involved!'); ?> <i class="fa fa-arrow-circle-right"></i>
+                                <?php echo $this->translate('Get involved!'); ?> <i
+                                    class="fa fa-arrow-circle-right"></i>
```

Comment:
This doesn't really look like the formatting is correct... Can you run `yarn format` again please?